docs/NodeDisruptionPolicy.md

```diff
@@ -0,0 +1,146 @@
+# NodeDisruptionPolicy
+
+Node Disruption policy allows the user to define to what actions to take for minor MachineConfig updates. The MCO is introducing this feature in TechPreview for 4.16.
```

Comment:
For additional context, it will be good idea to link enhancement https://github.com/openshift/enhancements/pull/1525

Reply:
Done (: